conda_smithy/cli.py

```diff
@@ -242,7 +242,7 @@ def __init__(self, parser):
         ]:
             scp.add_argument(
                 "--without-{}".format(ci.lower()),
-                dest=ci.lower(),
+                dest=ci.lower().replace("-","_"),
```

Comment:
This one is technically not needed but don't hurt either.

Reply:
yeah I thought we might as well plan ahead ;)